frontend/src/js/components/leftnav.js

```diff
@@ -28,7 +28,7 @@ import DocsLink from './common/docslink';
 
 const listItems = [
   { route: '/', text: 'Dashboard', canAccess },
-  { route: '/devices', text: 'Devices', canAccess: ({ userCapabilities: { canReadDevices } }) => canReadDevices },
+  { route: '/devices/accepted', text: 'Devices', canAccess: ({ userCapabilities: { canReadDevices } }) => canReadDevices },
```

Comment:
this should remain `/devices` as it is expected to be set by the previously selected state, if that wasn't selected before the devices should always default to `accepted`